.github/workflows/nodejs.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         env:
           GITHUB_CONTEXT: ${{ toJson(github) }}
         run: echo "$GITHUB_CONTEXT"
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
         with:
           fetch-depth: 1
```

Comment:
Thanks @ScottBrenner! This fetch-depth is no longer needed for v2 (it's the default) so you can remove this too.

Reply:
On it! https://github.com/digidem/mapeo-desktop/pull/285/commits/270b691533e57eb7223ebcdddaa8a589c1133164